test/functional/interface_rest.py

```diff
@@ -52,9 +52,7 @@ class RESTTest (BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 2
         self.extra_args = [["-rest", "-blockfilterindex=1"], []]
-        # whitelist peers to speed up tx relay / mempool sync
```

Comment:
edbc1ae33b0a17994331de0660b231e7f00a2bda

I like saving a few LOC, but dropping these comments here and there... Presumably they were helpful for the reader to understand why `whitelist`. And `noban_tx_relay` doesn't immediately mean speeding up.

Reply:
I agree it's helpful, but I am not sure about having it for all occurances. Perhaps we could put in `test_framework`, e.g.:

```diff
diff --git a/test/functional/test_framework/test_framework.py b/test/functional/test_framework/test_framework.py
index 220e51df38..aeb9a2691c 100755
--- a/test/functional/test_framework/test_framework.py
+++ b/test/functional/test_framework/test_framework.py
@@ -97,6 +97,7 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
         """Sets test framework defaults. Do not override this method. Instead, override the set_test_params() method"""
         self.chain: str = 'regtest'
         self.setup_clean_chain: bool = False
+        # whitelist peers to speed up tx relay / mempool sync
         self.noban_tx_relay: bool = False
         self.nodes: List[TestNode] 
```

Reply:
This could yield false positives: sometimes we need just this flags for a different reason than speeding up, right? I also don't think we can reliably expect a test code reader to figure out his answer ("why noban") is in the test framework.

Reply:
Fair enough, gonna add comments.